test/sequential/test-net-connect-local-error.js

```diff
@@ -3,28 +3,21 @@ const common = require('../common');
 const assert = require('assert');
 const net = require('net');
 
+const expectedErrorCodes = ['ECONNREFUSED', 'EADDRINUSE'];
```

Comment:
Format suggested is fine. `EADDRINUSE` doesn't occur every time, just sometimes, right? `EADDRINUSE is expected` makes it sound like it occurs every time. Maybe `EADDRINUSE can occur on FreeBSD.`?